include/mpi.h

```diff
@@ -160,6 +160,10 @@ enum {
     MPI_F_STATUS_SIZE   = 8
 };
 
+typedef struct {
+  int internal[MPI_F_STATUS_SIZE];
```

Comment:
Alternatively, we could just do:

```suggestion
  MPI_Status mpi_val;
```

which would be more alike to the way opaque handles are defined.